<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrightnessWindow</class>
 <widget class="QWidget" name="BrightnessWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>213</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Brightness Window</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string extracomment="label text which describes about changing brightness of the image">Brightness</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="lineBrightness">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="maxLength">
        <number>4</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSlider" name="sliderBrightness">
     <property name="minimum">
      <number>-100</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string extracomment="label text which describes about changing contrast of the image">Contrast</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="lineContrast">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="maxLength">
        <number>4</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSlider" name="sliderContrast">
     <property name="minimum">
      <number>-30</number>
     </property>
     <property name="maximum">
      <number>30</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string extracomment="label text which describes about changing gamma value of the image">Gamma Value</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="lineGamma">
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="inputMethodHints">
        <set>Qt::ImhDigitsOnly</set>
       </property>
       <property name="maxLength">
        <number>4</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QSlider" name="sliderGamma">
     <property name="minimum">
      <number>-30</number>
     </property>
     <property name="maximum">
      <number>30</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxForAll">
     <property name="text">
      <string>Change all images of the volume</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="resetButton">
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>162</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>sliderBrightness</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BrightnessWindow</receiver>
   <slot>onSliderBrightness_valueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderContrast</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BrightnessWindow</receiver>
   <slot>onSliderContrast_valueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sliderGamma</sender>
   <signal>valueChanged(int)</signal>
   <receiver>BrightnessWindow</receiver>
   <slot>onSliderGamma_valueChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>122</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resetButton</sender>
   <signal>clicked()</signal>
   <receiver>BrightnessWindow</receiver>
   <slot>onResetButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>116</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>181</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineBrightness</sender>
   <signal>textChanged(QString)</signal>
   <receiver>BrightnessWindow</receiver>
   <slot>onLineBrightness_textChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineContrast</sender>
   <signal>textChanged(QString)</signal>
   <receiver>BrightnessWindow</receiver>
   <slot>onLineContrast_textChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>72</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineGamma</sender>
   <signal>textChanged(QString)</signal>
   <receiver>BrightnessWindow</receiver>
   <slot>onLineGamma_textChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>182</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onSliderBrightness_valueChanged(int)</slot>
  <slot>onSliderContrast_valueChanged(int)</slot>
  <slot>onSliderGamma_valueChanged(int)</slot>
  <slot>onResetButton_clicked()</slot>
  <slot>onLineBrightness_textChanged(QString)</slot>
  <slot>onLineContrast_textChanged(QString)</slot>
  <slot>onLineGamma_textChanged(QString)</slot>
 </slots>
</ui>
